src/table/__integ__/inline-editing.test.ts

```diff
@@ -165,7 +165,7 @@ describe.each([false, true])('enableKeyboardNavigation=%s', enableKeyboardNaviga
         await page.keys(['Tab', 'Tab']);
         await page.keys(['ArrowDown', 'ArrowRight']);
       } else {
-        await page.keys(range(11).map(() => 'Tab'));
+        await page.keys(range(12).map(() => 'Tab'));
```

Comment:
Why this change?

Reply:
Because I made 2nd column sortable on the page - it made the header focusable, so an extra tab stop for when keyboard nav is not enabled.